public/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>RN Web</title>
+  <style>
+    #app-root {
+      display: flex;
+      flex: 1 1 100%;
+      height: 100vh;
+    }
+
+    @font-face {
+      font-family: 'SpaceGrotesk-Bold';
+      src: url('./assets/fonts/SpaceGrotesk-Bold.ttf') format('truetype'),
+    }
+
+    @font-face {
+      font-family: 'SpaceGrotesk-Light';
+      src: url('./assets/fonts/SpaceGrotesk-Light.ttf') format('truetype'),
+    }
+
+    @font-face {
+      font-family: 'SpaceGrotesk-Medium';
+      src: url('./assets/fonts/SpaceGrotesk-Medium.ttf') format('truetype'),
+    }
+
+    @font-face {
+      font-family: 'SpaceGrotesk-SemiBold';
+      src: url('./assets/fonts/SpaceGrotesk-SemiBold.ttf') format('truetype'),
+    }
```

Comment:
This may be a stupid question, but do other elements in the app have these font-families specified on their selectors?

Like:
```
html {
  font-family: 'SpaceGrotesk-Medium';
}

h1, h2, h3, h4, h5, h6 {
  font-family: 'SpaceGrotesk-Bold';
}
```

Reply:
No but everything I read on https://github.com/necolas/react-native-web issues board says to just add them to the shell and then specify via `fontFamily` from what I can tell. 

That's how Ignite expo web works, it uses the same typography file and everything. We didn't do any set up of HTML elements with styling